src/components/ui/date-range-picker.tsx

```diff
@@ -0,0 +1,65 @@
+"use client";
```

Comment:
nit: the `components/ui` folder is generally used for shadcn components, so could we move custom UI components to a different folder (maybe `components/ui/custom`)?